config/jobs/kubernetes/sig-release/release-branch-jobs/1.13.yaml

```
@@ -448,6 +458,9 @@ periodics:
     labels:
       preset-service-account: "true"
       preset-k8s-ssh: "true"
+    annotations:
+      testgrid-dashboards: sig-release-1.13-all, sig-release-1.13-blocking, sig-node-kubelet
```

if order of dashboards matters
```suggestion
      testgrid-dashboards: sig-release-1.13-blocking, sig-release-1.13-all, sig-node-kubelet
```